### DISCUSSION TITLE
Wifi icon are show the Wifi signal Live update

### DISCUSSION THREAD

**Discussion by mhkanon** [chris1111/Wireless-USB-OC-Big-Sur-Adapter#39] [Ideas]
Can I ask you it possible to add this type of system to update the wifi signal in the menu bar(see the attachment)? When the signal low it shows a low signal. When it high it shows a high signal. It updates live all time. At current it only shows the icon only. But not update the signal. Thank you.

![Screenshot 2021-03-18 at 7 55 20 AM](https://user-images.githubusercontent.com/24652688/111562176-ee36bc80-87bf-11eb-8d77-4fa7b0cd50f2.png)

**Comment by chris1111** [comment]
It is not possible because I have made a set of wifi Icon like you said but its not working in dark Mode for Big Sur and Catalina, the only way to showing a WIFI icon dark and light mode for Big Sur is to using this Icon
The original Realtek Icon works but it does not look like the native mac icon
